src/main/webapp/app/exam/liveExamExerciseUpdateService.ts

```diff
@@ -0,0 +1,14 @@
+import { Injectable } from '@angular/core';
+import { BehaviorSubject } from 'rxjs';
+
+@Injectable({ providedIn: 'root' })
+export class LiveExamExerciseUpdateService {
+    private examExerciseIdAndProblemStatementSource = new BehaviorSubject({ exerciseId: -1, problemStatement: '' });
+    currentExerciseIdAndProblemStatement = this.examExerciseIdAndProblemStatementSource.asObservable();
+
+    constructor() {}
+
+    updateLiveExamExercise(exerciseId: number, problemStatement: string) {
+        this.examExerciseIdAndProblemStatementSource.next({ exerciseId: exerciseId, problemStatement: problemStatement });
```

Comment:
I do not understand why writing `{ exerciseId: exerciseId, problemStatement: problemStatement }`
instead of `{ exerciseId, problemStatement }` is problematic.
Later in `exam-navigation-bar` (ngOnInit 53/54) I explicitly access these values via their "keys" not by [0], [1] which would seem more like magic numbers to me.
Can anyone explain to me why my code seems to be wrong and is actively changed by lint --fix, etc.?

Reply:
I think this is standard in JS/TS. That when the **property name** and **the assigned variable** are the same, the shorthand version is preferred.` { exerciseId, problemStatement }`

Reply:
Oh ok, thanks I will try it out because on my older branch it did not work that way.

Reply:
It does work now. (It really seems to be the case, that this js change happened between my old and new branch) 👍